src/main/scala/scala/swing/Publisher.scala

```diff
@@ -79,7 +79,7 @@ private[swing] trait LazyPublisher extends Publisher {
 
 import scala.ref._
 
-private[swing] trait SingleRefCollection[+A <: AnyRef] extends Iterable[A] { self =>
+private[swing] trait SingleRefCollection[/*+*/ A <: AnyRef] extends Iterable[A] { self =>
```

Comment:
is this a left-over?

Reply:
No, this is stronger enforcement by dotty. Just remove the entire commented part.